tests/collections/asr/test_label_datasets.py

```diff
@@ -11,19 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 import os
 
 import pytest
 import torch
 
 from nemo.collections.asr.data.audio_to_label import TarredAudioToClassificationLabelDataset
+from nemo.collections.asr.data.feature_to_label import FeatureToSeqSpeakerLabelDataset
+from nemo.collections.asr.parts.feature_loader import ExternalFeatureLoader
 from nemo.collections.asr.parts.features import WaveformFeaturizer
 from nemo.collections.common import tokenizers
 
 
 class TestASRDatasets:
     labels = ["fash", "fbbh", "fclc"]
+    unique_labels_in_seq = ['0', '1', '2', '3', "zero", "one", "two", "three"]
```

Comment:
are these labels uniq per session or for whole manifest?
while training speaker diarizer we train per session, doesn't this mix with remaining classes and spoil the relative number while training?

Reply:
It's fine. We use the parse to convert to relative speakers for each seq_label